reference/constraints/Url.rst

```diff
@@ -10,6 +10,7 @@ Validates that a value is a valid URL string.
 |                | - `protocols`_                                                      |
 |                | - `payload`_                                                        |
 |                | - `checkDNS`_                                                       |
+|                | - `dnsMessage`_
```

Comment:
We need a `|` character in this line to end the cell (at the same position as on the other lines).